src/components/BreadcrumbItemLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { BreadcrumbItem, BreadcrumbItemProps } from "@patternfly/react-core";
+import { Link } from 'react-router-dom';
+
+type BreadcrumbItemLinkProps = {
+  children: ReactNode;
+  to: string;
+} & BreadcrumbItemProps;
+
+function BreadcrumbItemLink({
+  children,
+  to,
+  ...rest
+}: BreadcrumbItemLinkProps): ReactElement {
+  function render({ className, ariaCurrent }) {
+    return (
+      <Link className={className} aria-current={ariaCurrent} to={to}>
+        {children}
+      </Link>
```

Comment:
Here is a pattern to avoid `render` prop and potential re-rendering problems, although soon to say what kind of integration testing for this project.

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/PatternFly/LinkShim/LinkShim.tsx